README.md

```diff
@@ -2,6 +2,6 @@
 NHLStenden year 1 period 4 project innovate
 
 Function of app. User selects drink, confirmation button comes up, if user selects yes drink will be prepared. 
-Frontend: Hakan Ege
-Backend: Bernardo
+Frontend: Hakan Ege and Iustin
+Backend: Bernardo, Yohan and Joey
```

Comment:
Mention first name and last name and make it a bit more professional. 

This project will look good on your future portfolio